test/JDBC/input/full_text_search/fts-contains-vu-verify.sql

```diff
@@ -0,0 +1,409 @@
+-- Full syntax of CONTAINS: https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/t-sql/queries/contains-transact-sql.md
+
+-- test basic CONTAINS use: ... CONTAINS(col_name, <simple_term>) ...
+-- <simple_term> ::= { word | "phrase" }
+EXEC fts_contains_vu_prepare_p1 'love'
+GO
+
+EXEC fts_contains_vu_prepare_p1 'other'
+GO
+
+EXEC fts_contains_vu_prepare_p1 'arts'
+GO
+
+EXEC fts_contains_vu_prepare_p1 'performing'
+GO
+
+EXEC fts_contains_vu_prepare_p1 'performance'
+GO
+
+EXEC fts_contains_vu_prepare_p1 'quick'
+GO
+
+EXEC fts_contains_vu_prepare_p1 'grow'
+GO
+
+EXEC fts_contains_vu_prepare_p1 'actually'
+GO
+
+EXEC fts_contains_vu_prepare_p1 'helped'
+GO
+
+EXEC fts_contains_vu_prepare_p1 'version'
+GO
+
+EXEC fts_contains_vu_prepare_p1 '"come back"'
+GO
+
+EXEC fts_contains_vu_prepare_p1 '"much of the"'
+GO
+
+EXEC fts_contains_vu_prepare_p1 '"due to"'
+GO
+
+EXEC fts_contains_vu_prepare_p1 '"per cent"'
+GO
+
+EXEC fts_contains_vu_prepare_p1 '"so-called"'
+GO
+
+
+EXEC fts_contains_vu_prepare_p1 '"stand up"'
+GO
+
+EXEC fts_contains_vu_prepare_p1 '"every month"'
+GO
+
+EXEC fts_contains_vu_prepare_p1 '"as a result"'
+GO
+
+EXEC fts_contains_vu_prepare_p1 '"in Australia"'
+GO
+
+EXEC fts_contains_vu_prepare_p1 '"daily news"'
+GO
+
+-- REST NOT SUPPORTED YET
+-- COMMENTED OUT
```

Comment:
Rather than commenting these out, I would prefer to leave them in, and just have the expected output reflect the current status of the tests. This will ensure that (1) we are erroring gracefully and not crashing and (2) we do not forget to re-enable these tests in the future

Reply:
Done